### ISSUE TITLE
Exception 0xc0000005 0x8 0x0 0x0

### ISSUE THREAD

**Issue by tordenflesk** [shilangyu/scoop-search#43]
```
Exception 0xc0000005 0x8 0x0 0x0
PC=0x0

runtime.asmstdcall()
        /opt/hostedtoolcache/go/1.21.5/x64/src/runtime/sys_windows_amd64.s:65 +0x75 fp=0x22fca0 sp=0x22fc80 pc=0x45fa35
rax     0x0
rbx     0x57a240
rcx     0x5ccd20
rdi     0x7fffffde000
rsi     0x22fea0
rbp     0x22fde0
rsp     0x22fc78
r8      0x0
r9      0x22fee0
r10     0x59eb78
r11     0x21
r12     0x22fec0
r13     0x1
r14     0x579be0
r15     0x0
rip     0x0
rflags  0x10293
cs      0x33
fs      0x53
gs      0x2b
```

Last working version: 1.3.1

**Comment by shilangyu** [comment]
Please post your buckets `scoop bucket list` and reproduction steps.

**Comment by tordenflesk** [comment]
```
❯ su scoop-search
Uninstalling 'scoop-search' (1.3.1).
Removing shim 'scoop-search.shim'.
Removing shim 'scoop-search.exe'.
Unlinking ~\scoop\apps\scoop-search\current
'scoop-search' was uninstalled.

❯ scoop install main/scoop-search
Installing 'scoop-search' (1.3.3) [64bit] from main bucket
Starting download with aria2 ...
Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 85502b|OK  |   7.2MiB/s|C:/Users/Tordenflesk/scoop/cache/scoop-search#1.3.3#https_github.com_shilangyu_scoop-search_releases_download_v1.3.3_scoop-search.exe
Download: Status Legend:
Download: (OK):download completed.
Checking hash of scoop-search.exe ... ok.
Linking ~\scoop\apps\scoop-search\current => ~\scoop\apps\scoop-search\1.3.3
Creating shim for 'scoop-search'.
'scoop-search' (1.3.3) was installed successfully!
Notes
-----
To replace built-in scoop search, add this to $PROFILE: Invoke-Expression (&scoop-search --hook)

❯ scoop-search git
Exception 0xc0000005 0x8 0x0 0x0
PC=0x0

runtime.asmstdcall()
        /opt/hostedtoolcache/go/1.21.5/x64/src/runtime/sys_windows_amd64.s:65 +0x75 fp=0x22fca0 sp=0x22fc80 pc=0x45fa35
rax     0x0
rbx     0x57a220
rcx     0x5ccd00
rdi     0x7fffffdd000
rsi     0x22fea0
rbp     0x22fde0
rsp     0x22fc78
r8      0x0
r9      0x22fee0
r10     0x59eb58
r11     0x21
r12     0x22fec0
r13     0x1
r14     0x579bc0
r15     0x0
rip     0x0
rflags  0x10293
cs      0x33
fs      0x53
gs      0x2b

 ~  2 
❯
```

```
❯ scoop bucket list

Name                   Source                                             Updated             Manifests
----                   ------                                             -------             ---------
anderlli0053_DEV-tools https://github.com/anderlli0053/DEV-tools.git      14.01.2024 17:31:05     23810
Base                   https://github.com/shovel-org/Base                 13.01.2024 21:09:46       346
emulators              https://github.com/borger/scoop-emulators.git      14.01.2024 19:05:16        55
extras                 https://github.com/ScoopInstaller/Extras           14.01.2024 19:27:51      1904
games                  https://github.com/Calinou/scoop-games             14.01.2024 17:27:43       294
java                   https://github.com/ScoopInstaller/Java             14.01.2024 13:26:51       267
lemon                  https://github.com/hoilc/scoop-lemon               14.01.2024 19:17:27       989
main                   https://github.com/ScoopInstaller/Main             14.01.2024 17:26:39      1283
nerd-fonts             https://github.com/matthewjberger/scoop-nerd-fonts 10.01.2024 15:16:57       327
nirsoft                https://github.com/kodybrown/scoop-nirsoft         11.01.2024 22:17:03       276
nonportable            https://github.com/TheRandomLabs/scoop-nonportable 14.01.2024 20:02:34        80
versions               https://github.com/ScoopInstaller/Versions         14.01.2024 17:33:04       435
```
```
❯ si scoop-search@1.3.1
WARN  Given version (1.3.1) does not match manifest (1.3.2)
WARN  Attempting to generate manifest for 'scoop-search' (1.3.1)
Autoupdating scoop-search
Downloading scoop-search.exe to compute hashes!
scoop-search.exe (2,2 MB) [====================================================================] 100%
Computed hash: 47d91772ed506ce9c2fa46019db083dfd6fc542580dfe282841133fd5440fe5a
Writing updated scoop-search manifest
Installing 'scoop-search' (1.3.1) [64bit]
Loading scoop-search.exe from cache.
Checking hash of scoop-search.exe ... ok.
Linking ~\scoop\apps\scoop-search\current => ~\scoop\apps\scoop-search\1.3.1
Creating shim for 'scoop-search'.
'scoop-search' (1.3.1) was installed successfully!
Notes
-----
To replace built-in scoop search, add this to $PROFILE: Invoke-Expression (&scoop-search --hook)

❯ scoop-search git
'anderlli0053_DEV-tools' bucket:
    adobe-digital-editions (4.5.12)
    biodiff (1.1.0) --> includes 'git-biodiff.exe'
    clang@9.0.0 (9.0.0) --> includes 'git-clang-format'
    cmwtat (2.5.0.0) --> includes 'CMWTAT_Digital_Release_2_5_0_0.exe'
```

**Comment by shilangyu** [comment]
Is there any chance that the anti virus is killing the process? It seems to be plaguing me last few weeks

**Comment by shilangyu** [comment]
This should be resolved with [v1.4.0](https://github.com/shilangyu/scoop-search/releases/tag/v1.4.0).
